### ISSUE TITLE
add a toggle button for camera on mobile; rather than settings

### ISSUE THREAD

**Issue by steveseguin** [steveseguin/obsninja#730]
via AJ

**Comment by steveseguin** [comment]
yup; front/rear..  I think I see the value here, but not sure how much room there is on screen.  Cycling thru camera's is one option as well; might run into more issues here though.

**Comment by jcalado** [comment]
There's probably still room on mobile to get another button in, no?
Only on mobile though.

**Comment by steveseguin** [comment]
Closing old issue automatically by bot; mobile camera toggle now exists.
